src/main/java/school/lemon/changerequest/java/vehicles/IBoatImpl.java

```diff
@@ -0,0 +1,91 @@
+package school.lemon.changerequest.java.vehicles;
+
+/**
+ * Created by lera on 08.12.16.
+ */
+public class IBoatImpl extends Vehicle implements IBoat {
+
+    private boolean motorized;
+
+    public IBoatImpl(String make, int manufacturedYear, String model, boolean motorized) {
+        super(make, manufacturedYear, model);
+        this.motorized = motorized;
+    }
+
+
+    @Override
+    public boolean isMotorized() {
+        return motorized == true;
+    }
+
+    @Override
+    public void setMotorized(boolean motorized) {
+        this.motorized = motorized;
+    }
+
+    @Override
+    public int getManufacturedYear() {
+        return manufacturedYear;
+    }
+
+    @Override
+    public void setManufacturedYear(int year) {
+        this.manufacturedYear = year;
+
+    }
+
+    @Override
+    public String getMake() {
+        return make;
+    }
+
+    @Override
+    public void setMake(String make) {
+        this.make = make;
+    }
+
+    @Override
+    public String getModel() {
+        return model;
+    }
+
+    @Override
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    @Override
+    public String accelerate() {
+        return "jet water";
+    }
+
+    @Override
+    public String steerLeft() {
+        return "turn tiller left";
+    }
+
+    @Override
+    public String steerRight() {
+        return "turn tiller right";
+    }
+
+    public String printismotorized() {
```

Comment:
1) this method should be private (as no one will use it, except your own class)
2) method name should follow Java Code Convention - lowerCamelCase: `printIsMotorized`
3) you can remove this method and use ternary `if-else` statement: ` isMotorized() ? " (with motor)." : "."` directly in `toString` method